docker-compose.yml

```diff
@@ -1,15 +1,51 @@
 version: '3.8'
 
 services:
+  
+  postgres:
+    image: postgres:17
+    container_name: postgres-db
+    restart: unless-stopped
+    environment:
+      POSTGRES_USER: myuser
+      POSTGRES_PASSWORD: mypassword
+      POSTGRES_DB: mydatabase
```

Comment:
We probably should change the environment username, password, and database name to be defined in a `.env` file that isn't committed to git but this is fine for now I think.